### ISSUE TITLE
SBCL 1.2.9 not supported?

### ISSUE THREAD

**Issue by kkaos** [mabragor/cl-read-macro-tokens#3]
Using Quicklisp and SBCL 1.2.9, I get the following error when trying to load cl-read-macro-tokens:

This version of SBCL is not supported!

I'm really wanting to use cl-yaclyaml, but that won't load because this is a dependency.  What's the latest version of SBCL cl-read-macro-tokens supports?

Thank you for your time!

**Comment by mabragor** [comment]
Hi!

The thing is, cl-read-macro tokens relies on replacing read-list function (which is sbcl internal,
so not guarded by any standards). And in last releases of sbcl the details of the function
seem to change with every release. Changes to support the new version are usually small,
yet have to be done manually, so I'll look into it and see what I can do.

**Comment by kkaos** [comment]
Hello,

Thanks for the reply.  Supporting SBCL sounds frustrating.  Which Common
Lisp implementation do you typically use?

**Comment by mabragor** [comment]
It's SBCL, only not the latest release, but rather 1.2.4, which is 
supplied with Debian Wheezy.

**Comment by mabragor** [comment]
Hmm, I've just checked and for me everything seems to work.
I've compiled SBCL from git (tag sbcl-1.2.9),
did (ql:update-all-dists) and then
(ql:quickload 'cl-read-macro-tokens)
(ql:quickload 'cl-read-macro-tokens-tests)
(cl-read-macro-tokens:run-tests)

Ditto, tests for cl-yaclyaml work as well

Maybe, you are using master version of sbcl and not the tagged one?

**Comment by kkaos** [comment]
No, the problem is the fact that I'm a newbie with quicklisp; I wasn't
aware that I should be running (ql:update-all-dists) to keep my quicklisp
packages up-to-date.  After doing so, both cl-read-macro-tokens and
cl-yaclyaml load for me.

Thanks for your help, and sorry about the unnecessary bug report.  Looking
forward to using cl-yaclyaml!

**Comment by PuercoPop** [comment]
Instead of updating to each implementation's quasiquote object wouldn't it be a better idea to replace the implementation's quasiquote representation for a list based one, such as [fare-quasiquote](https://gitlab.common-lisp.net/frideau/fare-quasiquote/)?
